API.md

```diff
@@ -162,6 +201,44 @@ An `HTTP 404 Not Found` status code is returned if the case group with the speci
 
 An `HTTP 404 Not Found` status code is returned if the case with the specified ID could not be found.
 
+## Find Cases by Sample Unit Id
+* `GET /cases/sampleunitids` will return the details of the case with the given sample unit ID.
+
+### Example JSON Response
+```json
+{
+  [
+    {
+    "state":"ACTIONABLE",
```

Comment:
This indentation is still off. this section need to come further across from the brace above